lib/util/template/index.ts

```diff
@@ -121,11 +121,11 @@ export const allowedFields = {
   newDigestShort:
     'A shorted version of newDigest, for use when the full digest is too long to be conveniently displayed',
   newMajor:
-    'The major version of the new version. e.g. "3" if the new version if "3.1.0"',
+    'The major version of the new version. e.g. "3" if the new version is "3.1.0"',
```

Comment:
Please double check if the PR title is correct. I don't know how specific you want me to be:

- `refactor: fix typo`
- `refactor(template): fix typo`
- `refactor(util/template): fix typo`
- `refactor(lib/util/template): fix typo`